frontend/src/components/eventsModal.js

```diff
@@ -0,0 +1,94 @@
+import { Component } from 'react'
+import Router from 'next/router'
+import {
+  Alert,
+  Button,
+  Form,
+  FormGroup,
+  Label,
+  Input,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter
+} from 'reactstrap'
+import { createEvent } from '../utils/api'
+
+class EventsModal extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      alert: ''
+    }
+  }
+
+  addEvent = async () => {
+    const event = {
+      name: this.state.name ? this.state.name : '',
+      date: this.state.date ? this.state.date : '',
+      startTime: this.state.startTime ? this.state.startTime : '',
+      endTime: this.state.endTime ? this.state.endTime : '',
+      location: this.state.location ? this.state.location : '',
+      description: this.state.description ? this.state.description : '',
+      fbLink: this.state.fbLink ? this.state.fbLink : ''
+    }
+    const { code } = await createEvent(event)
+    if (code === 500) {
```

Comment:
use the `success` field of the response not the code (also 500 should not map to a user's error). This means that you should be checking fields in backend too!